aws/resource_aws_fsx_lustre_file_system_test.go

```diff
@@ -24,49 +24,52 @@ func init() {
 
 func testSweepFSXLustreFileSystems(region string) error {
 	client, err := sharedClientForRegion(region)
+
 	if err != nil {
 		return fmt.Errorf("error getting client: %s", err)
 	}
+
 	conn := client.(*AWSClient).fsxconn
+	sweepResources := make([]*testSweepResource, 0)
+	var errs *multierror.Error
 	input := &fsx.DescribeFileSystemsInput{}
 
-	err = conn.DescribeFileSystemsPages(input, func(page *fsx.DescribeFileSystemsOutput, lastPage bool) bool {
+	err = conn.DescribeFileSystemsPages(input, func(page *fsx.DescribeFileSystemsOutput, isLast bool) bool {
```

Comment:
Nit: Prevalent convention for anonymous paginator functions is to name the 2nd parameter `lastPage`

Reply:
Oh no!!! I was just going on the contributing docs and changed all the WAFs the wrong way!